nexus/reconfigurator/planning/src/system.rs

```diff
@@ -344,11 +350,8 @@ impl SystemDescription {
         for sled in self.sleds.values() {
             let sled_details = SledDetails {
                 policy: sled.policy,
-                state: SledState::Active,
```

Comment:
(Musing, no action necessary)

This feels like the crux of this PR to me -- converting the system description to a `PlanningInputBuilder` no longer makes assumptions about the sled state being active, but lets that be a parameter instead.

Reply:
Yep!